src/js/components/Pagination/__tests__/Pagination-test.tsx

```diff
@@ -456,13 +456,15 @@ describe('Pagination', () => {
     window.scrollTo = jest.fn();
     const user = userEvent.setup();
     const App = () => {
-      const [step, setStep] = useState(10);
+      const [step, setStep] = useState<number>(10);
       return (
         <Grommet>
           <Select
             a11yTitle="Controlled select"
             options={[10, 25, 50, 100]}
-            onChange={({ option }) => setStep(option)}
+            onChange={({ option }) => {
+              if (typeof option === 'number') setStep(option);
```

Comment:
why did we add this?

Reply:
I added that because OnChange will accept an argument of type OptionType, and option is a type of number, if we don't check the type, it will show an error.

Reply:
I don't think we need to adjust this test -- it looks like the test has all options defined as numbers, so I don't think an error will be thrown. Let's keep the PR lean if possible.